app-misc/fhem/fhem-5.8.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+DESCRIPTION="A GPL'd perl server for house automation"
+HOMEPAGE="https://www.fhem.de/"
+SRC_URI="https://www.fhem.de/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="doc"
+
+RDEPEND="dev-perl/Crypt-CBC
+	dev-perl/Device-SerialPort
+	dev-perl/Digest-CRC
+	dev-perl/JSON"
+
+DEPEND="media-gfx/pngcrush"
+
+DOCS=( "CHANGED" "HISTORY" "README.SVN" "README_DEMO.txt" "docs"/*.txt "docs"/*.patch "docs"/*.pdf "docs/changelog" "docs/copyright" "docs/dotconfig" "docs/fhem.odg.readme" "docs/LIESMICH.update-thirdparty" "docs"/README* "docs/X10" )
+
+QA_PREBUILT="opt/fhem/contrib/lcd4linux/fritzbox_dpf/lcd4linux
+	opt/fhem/contrib/lcd4linux/rpi_dpf/lcd4linux"
+
+pkg_setup() {
+	enewgroup fhem
+	enewuser fhem -1 -1 /opt/fhem fhem
+}
+
+src_prepare() {
+	default
+
+	cp "${FILESDIR}"/fhem.cfg fhem.cfg || die
+
+	# Fix install path in Makefile
+	sed -i -e 's,^\(BINDIR=\).*,\1'${D}/opt/fhem',' Makefile || die
```

Comment:
Don't sed paths in, this is fragile on characters used. Instead, put `$(DESTDIR)` into the Makefile and use it.

Reply:
Okay, changed.